locales/en.default.json

```
@@ -44,6 +44,16 @@
       "click_to_log_in": "Please click here to log in"
     }
   },
+  "breadcrumbs": {
```

Add these lines into tags section
```suggestion
 "tags": {
    "__outline": {
      "label": "Categories"
    },
```